shared/feature/feeding/src/main/java/com/epmedu/animeal/feeding/domain/model/Feeding.kt

```diff
@@ -1,11 +1,11 @@
 package com.epmedu.animeal.feeding.domain.model
 
+import com.epmedu.animeal.users.domain.model.User
 import java.util.Date
 
 data class Feeding(
     val id: String,
-    val name: String,
-    val surname: String,
+    val feeder: User?,
```

Comment:
is it possible to make this non-nullable?

Reply:
It is intended, because user can be removed, but we want to show his feedings